src/Decoder.c

```diff
@@ -4804,8 +4804,12 @@ static ZyanStatus ZydisDecodeInstruction(ZydisDecoderContext* context,
                         ZYAN_MEMCPY(&instruction->accessed_flags, &flags->action,
                             sizeof(flags->action));
 
-                        instruction->flags_read = flags->flags_read;
-                        instruction->flags_written = flags->flags_written;
+                        instruction->cpu_flags_read = flags->flags_read & 0x003FFFFF;
```

Comment:
Not a fan of the hardcoded mask. Can we do `(1 << MAX_FLAG) - 1` here?

Reply:
This mask will be obsolete after doing the generator change.